config-templates/ibm/ibm-object-storage-plugin/2.2/custom-parameters.json

```diff
@@ -0,0 +1,42 @@
+[
+    {
+        "description": "Release name of the chart",
+        "displayname": "Helm Chart Release Name",
+        "name": "helm-release-name",
+        "category": "config",
+        "required": "true"
+    },
+    {
+        "description": "Helm Chart Additional Parameters (Optional)",
+        "displayname": "Helm Chart Additional Parameters (Optional)",
+        "name": "parameters",
+        "category": "config",
+        "required": "false",
+        "type": "csv",
+        "mutable": "true"
+    },
+    {
+        "description": "COS License (Apache 2.0)",
+        "displayname": "COS License (Apache 2.0)",
```

Comment:
Update `description` and `displayname` as
 "COS plug-in License: Apache License Version 2.0"